src/reducers/versions.tsx

```diff
@@ -339,7 +340,8 @@ export type VersionsState = {
   };
   currentBaseVersionId:
     | number // data successfully loaded
-    | undefined; // data has not yet loaded
+    | undefined // data has not yet loaded
+    | false; // data loaded but it is empty
```

Comment:
I find this comment confusing. By "data loaded but it is empty", do you mean an ID has been loaded for baseVersion, but the baseVersion itself has not been loaded? (I think that's what `false` means). To me that's not what the comment suggests. It actually sounds like the data has been loaded and also has not been loaded (because it's empty), which is what I find confusing.

Reply:
I updated the comments to make it more clear.